bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/relay.xml

```diff
@@ -284,4 +262,217 @@
 			<channel id="powerLed" typeId="ledPowerDisable" />
 		</channels>
 	</channel-group-type>
+	<channel-group-type id="externalSensors">
+		<label>External Sensors</label>
+		<description>Temperatures from external sensors connected to the optional Addon</description>
+	</channel-group-type>
+	<channel-type id="timerAutoOn" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Auto-ON Timer</label>
+		<description>ON: After the output was turned off it turns on automatically after xx seconds; 0 disables the timer</description>
+		<state min="0" step="1" pattern="%d %unit%" readOnly="false"></state>
+	</channel-type>
+	<channel-type id="timerAutoOff" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Auto-OFF Timer</label>
+		<description>ON: After the output was turned on it turns off automatically after xx seconds; 0 disables the timer</description>
+		<state min="0" step="1" pattern="%d %unit%" readOnly="false"></state>
+	</channel-type>
+	<channel-type id="timerActive" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Auto ON/OFF timer active</label>
+		<description>ON: A timer is active, OFF: no timer active</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="rollerShutter">
+		<item-type>Rollershutter</item-type>
+		<label>Roller Control (0=open, 100=closed))</label>
+		<description>Controls the roller</description>
+		<category>Blinds</category>
+	</channel-type>
+	<channel-type id="rollerPosition" advanced="true">
```

Comment:
Why are rollerPosition and rollerShutter two different channels?

Reply:
because OH and Shelly API use inverted position values
OH: 0=CLOSED, 100=OPEN
Shelly: 100=CLOSED, 0=OPEN
(or vice versa)
There are UI widgets, which require the inverted value

Reply:
So rollerPosition and rollerShutter are inverted from eachother?

Reply:
yes, e.g. wenn rollerPos is 20 the shutterControl has the value of 80

Reply:
I encountered the same problem in my own binding, I just made a boolean configuration that allowed the user to choose to have the percentage values inverted or not. I doubt users care what shelly actually thinks the values are, they just want integration with openhab to be seamless. I'd rather not confuse them with multiple channels that are transforms of each other.

Reply:
this is in since the initial release and one of the users: myself
the channel is advanced, so not visible by defualt
and documented in the README
A boolean config option makes no sense, because it breaks one way or the other.

Reply:
I'm not sure what you mean by breaking. A rollershutter item accepts UpDownType, PercentType, and StopMoveType. It does not accept an OpenClosedType. What you might be caught up on is how openhab converts an OpenClosedType to a PercentType which is indeed 0=CLOSED, 100=OPEN. But a rollershutter doesn't support a OpenClosedType so where is the issue? Is openhab converting it before it reaches the rollershutter item or are you doing it in the binding?
It should entirely be the binding's prerogative as to what it means to be opened/closed or how to interpret a percent command.

How do you expect users to use your two channels? Would they use the rollershutter item for up/down/stop commands but send all percent commands to the dimmer instead? Would users ever send percent commands to both the dimmer and rollershutter? Would the percentage values from both channels ever be used at the same time?

Reply:
uff, both channels work in sync
I could send 30 to the RollerShutter, which is synced up to 70 for the rollerPosition
and vice versa. It's just the reverted value to put/get depending what the UI control needs.